tests/block_tools.py

```diff
@@ -650,7 +656,7 @@ def get_consecutive_blocks(
                         latest_block = blocks[full_block.header_hash]
                         finished_sub_slots_at_ip = []
                         num_blocks -= 1
-                        if num_blocks == 0:
+                        if num_blocks <= 0 and not keep_going_until_tx_block:
```

Comment:
This return condition would be a bit easier to read as something like `if done:` with done being set above, but I see `keep_going_until_tx_block` is used elsewhere. No action needed.